Tests/TestBase.py

```diff
@@ -3,6 +3,8 @@
 import pathlib
 import sys
 from collections import deque
+import cProfile
```

Comment:
assuming I don't already have this dependency and the one below, make sure to add to requirements.txt

Reply:
cProfile is part of cpython distribution

Reply:
I should actually remove that code, that was just for my tests